src/main.rs

```diff
@@ -359,6 +360,48 @@ fn gcs_storage_backend(log: &Logger, m: &clap::ArgMatches) -> Result<VfsProducer
     }))
 }
 
+fn azblob_storage_backend(log: &Logger, m: &clap::ArgMatches) -> Result<VfsProducer, String> {
+    let mut b = opendal::services::Azblob::default();
+    if let Some(val) = m.value_of(args::AZBLOB_ROOT) {
+        b.root(val);
+    }
+    if let Some(val) = m.value_of(args::AZBLOB_CONTAINER) {
+        b.container(val);
+    }
+    if let Some(val) = m.value_of(args::AZBLOB_ENDPOINT) {
+        b.endpoint(val);
+    }
+    if let Some(val) = m.value_of(args::AZBLOB_ACCOUNT_NAME) {
+        b.account_name(val);
+    }
+    if let Some(val) = m.value_of(args::AZBLOB_ACCOUNT_KEY) {
+        b.account_key(val);
+    }
+    if let Some(val) = m.value_of(args::AZBLOB_SAS_TOKEN) {
+        b.sas_token(val);
+    }
+    if let Some(val) = m.value_of(args::AZBLOB_BATCH_MAX_OPERATIONS) {
+        b.batch_max_operations(
+            val.parse::<usize>().map_err(|e| {
+                format!("could not parse AZBLOB_BATCH_MAX_OPERATIONS to usize: {e}")
+            })?,
+        );
+    }
+    let accessor = b
+        .build()
+        .map_err(|e| format!("could not build Azblob: {e}"))?;
+    let op = opendal::OperatorBuilder::new(accessor).finish();
```

Comment:
Better to use [`Operator::new`](https://docs.rs/opendal/0.47.1/opendal/struct.Operator.html#method.new) instead.